src/main/features/response/views/confirmation/part-admission-steps.njk

```diff
@@ -88,4 +88,19 @@
 
 {% endif %}
 
+{% if (claim.response.paymentIntention === undefined ) %}
+  <!-- States Paid -->
+  <h3 class="bold">{{ t('If {{ claimantName }} accepts your response', { claimantName: claimantName }) }}</h3>
+  <p>The claim will be settled.</p>
+  <h3 class="bold">{{ t('If {{ claimantName }} rejects your response', { claimantName: claimantName }) }}</h3>
+  {%  if claim.response.freeMediation === FreeMediationOption.YES %}
+    <p>{{ t('We’ll ask if they want to try mediation. If they agree, we’ll contact you with an appointment.') }}</p>
+    <p>{{ t('If they don’t want to try mediation the court will review the case. You might have to go to a hearing.') }}</p>
+    <p>{{ t('We`ll contact you to tell you what to do next.') }}</p>
+  {% else %}
+    <p>{{ t('The court will review the case. You might have to go to a hearing.') }}</p>
+    <p>{{ t('We`ll contact you to tell you what to do next.') }}</p>
```

Comment:
This is the wrong kind of apostrophe; line 102 and 99 at time of writing.

Reply:
@andrewwa-kainos  have fixed it now. Thanks